frontend/src/modules/user/templates/login.hbs

```diff
@@ -12,7 +12,13 @@
                     <i class="fa fa-times"></i><span id="loginErrorMessage"/>
                 </div>
                 <input type="email" autocomplete="off" class="login-input-username" value="" id="login-input-username" placeholder="{{t 'app.usernameoremail'}}" tabindex="1" />
-                <input type="password" autocomplete="off" class="login-input-password" value="" id="login-input-password" placeholder="{{t 'app.password'}}" tabindex="2" />
+                <div class="login-password-container">                
+                  <input type="password" autocomplete="off" class="login-input-password" value="" id="login-input-password" placeholder="{{t 'app.password'}}" tabindex="2" />
+                  <button aria-pressed="false" class="login-password-reveal">
+                    <svg class="login-password-reveal-show" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M0 0h24v24H0zm0 0h24v24H0zm0 0h24v24H0zm0 0h24v24H0z" fill="none"/><path d="M12 7c2.76 0 5 2.24 5 5 0 .65-.13 1.26-.36 1.83l2.92 2.92c1.51-1.26 2.7-2.89 3.43-4.75-1.73-4.39-6-7.5-11-7.5-1.4 0-2.74.25-3.98.7l2.16 2.16C10.74 7.13 11.35 7 12 7zM2 4.27l2.28 2.28.46.46C3.08 8.3 1.78 10.02 1 12c1.73 4.39 6 7.5 11 7.5 1.55 0 3.03-.3 4.38-.84l.42.42L19.73 22 21 20.73 3.27 3 2 4.27zM7.53 9.8l1.55 1.55c-.05.21-.08.43-.08.65 0 1.66 1.34 3 3 3 .22 0 .44-.03.65-.08l1.55 1.55c-.67.33-1.41.53-2.2.53-2.76 0-5-2.24-5-5 0-.79.2-1.53.53-2.2zm4.31-.78l3.15 3.15.02-.16c0-1.66-1.34-3-3-3l-.17.01z"/></svg>
```

Comment:
It looks like the user profile already has the ability to unmask password input. The user profile uses the font awesome eye icon. Please could you use the same thing here for consistency?

Reply:
Is the position ok? Or would you rather have the eye outside the input (alternatively, I could user profile's version to be within the input)?

Reply:
I am happy with the position, I have seen other sites that do this and position the eye inside the password field. I think it would make sense to move the user profile eye icon inside the box as well for consistency if you don't mind.

I'm not sure if you've tested this on Edge, but just so that you're aware it has its own password reveal icon that it puts inside password fields - you may need to hide this on edge so that they don't interfere with each other.